corebehrt/data/prepare_data.py

```diff
@@ -219,7 +220,7 @@ def _prepare_mlm_features(self, predefined_splits, val_ratio=0.2) -> Data:
 
         # 5. Truncation
         logger.info(f"Truncating data to {data_cfg.truncation_len} tokens")
-        data = truncate_data(data, data_cfg.truncation_len, vocab, truncate_patient)
+        data = self._truncate_data(data, vocab, data_cfg)
```

Comment:
Don't exactly remember what we decided here, but I believe we shouldn't pass `data_cfg` around and instead only pass in the relevant elements of `data_cfg` into the function (In this case `priority_truncation` and `truncation_len`